gui/src/settingsdialog.cpp

```diff
@@ -149,6 +151,21 @@ SettingsDialog::SettingsDialog(Settings *settings, QWidget *parent) : QDialog(pa
 	audio_buffer_size_edit->setPlaceholderText(tr("Default (%1)").arg(settings->GetAudioBufferSizeDefault()));
 	connect(audio_buffer_size_edit, &QLineEdit::textEdited, this, &SettingsDialog::AudioBufferSizeEdited);
 
+	audio_device_combo_box = new QComboBox(this);
+	QList<QString> audio_out_device_list = {};
+	for (QAudioDeviceInfo di : QAudioDeviceInfo::availableDevices(QAudio::AudioOutput))
+	{
+		audio_out_device_list.append(di.deviceName());
```

Comment:
No need, for the temporary list, just use `audio_device_combo_box->addItem(di.deviceName(), di.deviceName())` (note the second arg)
Also remove space after `for` and unnecessary `{`s.

Reply:
done